org-code-javabuilder/theater/src/main/java/org/code/theater/Stage.java

```diff
@@ -107,16 +109,21 @@ public void playSound(String filename) throws FileNotFoundException {
    *     implemented using an array of loopable sounds (Mike can generate these).
    */
   public void playNote(Instrument instrument, int note, double seconds) {
-    final String sampleFilePath = instrumentSampleLoader.getSampleFilePath(instrument, note);
-    if (sampleFilePath == null) {
-      return;
-    }
+    this.playNote(instrument, note, seconds, false);
+  }
 
-    try {
-      this.audioWriter.writeAudioFromLocalFile(sampleFilePath, seconds);
-    } catch (FileNotFoundException e) {
-      System.out.printf("Could not play instrument: %s at note: %s%n", instrument, note);
-    }
+  /**
+   * Plays a note with the selected instrument and adds a pause for the duration of the note.
+   * Convenience method for playing notes in sequence without needing to call pause() between each
+   * one.
```

Comment:
Does the pause cause a pause of the note's length after the note plays, or 'pause' the length of the note starting at when the note starts playing, so there is no audible pause between notes if you always called `playNoteAndPause`?

Reply:
The second case, yeah - if you called `playNoteAndPause` with a time of 4 seconds, you'd hear the note play for 4 seconds and the next play command would play audio right after the note finished playing with no audible pause between. In contrast, multiple calls to `playNote` (and `playSound`) will play sounds at the same time unless pause is called in between.

Reply:
Makes sense. Can you clarify that in the comment?